.github/workflows/gitlab.yml

```diff
@@ -11,7 +11,7 @@ permissions: {}
 jobs:
   trigger-gitlab-pipeline:
     runs-on: [self-hosted, gitlab]
-    if: ${{ github.event_name == 'push' || (github.event_name == 'pull_request_target' && contains(github.event.pull_request.labels.*.name, 'run tests')) }}
+    if: ${{ github.event_name == 'push' || (github.event_name == 'pull_request_target' && contains(github.event.pull_request.labels.*.name, 'gitlab')) }}
```

Comment:
True that this doesn't exactly reflect what is happening under the hood, but would there really be a scenario where you would want to trigger tests but not gitlab, or vice versa? i would maybe lean towards both to be triggered by one label

Reply:
I'd say the test should be run in all cases like it was (almost) before. The gitlab CI runs only when we (admins) attach a label to the PR.

Reply:
do you mean tests should be ran for everyone, regardless of the label?

Reply:
Yes. Do you have an argument against it?

Reply:
Since the tests are not running on our internal runners, i guess yeah, don't see why we can't run these automatically

Reply:
Specially given that the analytics tests portion didn't receive the same treatment here :D